### PR TITLE
why the fuck do fuzzycuffs act as standards

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
fuzzycuffs are now 3 seconds to break out of instead of 120
fuzzy legcuffs are now 3 seconds to break out of instead of 30 or whatever
## Changelog
:cl:
tweak: Fuzzy handcuffs, like the ones you kinky bastards use, are now 3 seconds to break out of instead of 120. Who thought this through? Clearly, someone didn't.
tweak: Fuzzy legcuffs take 3 seconds to break out of instead of 30. Yikes, bro.
/:cl:

### CODE REVIEW

**Comment by Keekenox** [comment]
HOW DARE YOU HATTER, HOW AM I SUPPOSED TO USE FUZZY HANDCUFFS AS A SECURITY OFFICER NOW?! DELETE PR DELETE PR DELETE PR REEEEEEEEEEEEEE

**Comment by Knouli** [comment]
Might want to change fuzzy leg cuffs too, they act as standard at 30 seconds to breakout. |

https://github.com/Citadel-Station-13/Citadel-Station-13-RP/blob/387ef4b95ed242fe9a12ae4d311e0a4e05d8c984/code/game/objects/items/weapons/handcuffs_vr.dm

File: code/game/objects/items/weapons/handcuffs.dm [hunk 1]
```diff
@@ -141,6 +141,7 @@ var/last_chew = 0
 	name = "fuzzy cuffs"
 	icon_state = "fuzzycuff"
 	desc = "Use this to keep... 'prisoners' in line."
+	breakkouttime = 30 //3sec breakout time. why did this not exist before. bruh moment.
```
Comment (TheFurryFeline) [inline code/game/objects/items/weapons/handcuffs.dm]: Extra k's making Travis screech here.

Comment (Hatterhat) [reply code/game/objects/items/weapons/handcuffs.dm]: when the fuck did you start paying attention to this codebase

**Pull request merged**